opstack/src/builder.rs

```diff
@@ -92,7 +92,16 @@ impl OpStackClientBuilder {
         );
         let consensus = ConsensusClient::new(&config);
         let fork_schedule = ForkSchedule {
```

Comment:
I know since right now we only have configs for mainnet opstack chains these configs are all identical, but its possible we will have some testnet chains, or even some chains that don't follow the superchain hardfork schedule in the future. We might want to support setting this value from the config then, similar to what we do for Ethereum.

Reply:
makes sense, any change required for now ?

Reply:
Yeah I think just for completeness, lets add the fork schedule to the opstack config, and then we can just have a `SuperchainForkSchedule::mainnet` function (and at some point we can add testnets as needed) just like how we have `EthereumForkSchedule::mainnet` in the eth crate.